mcs/class/Facades/netstandard/TypeForwarders.cs

```diff
@@ -2446,7 +2446,7 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Diagnostics.Tracing.EventSourceCreatedEventArgs))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Drawing.KnownColor))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Data.Common.DbProviderFactories))]
-#if SYSTEM_DRAWING
+#if !NO_SYSTEM_DRAWING
```

Comment:
I think this should point to System.Drawing.Common instead in all cases. @akoeplinger thoughts?